Makefile

```diff
@@ -304,6 +304,7 @@ container_composer_golangci_built.info: Makefile Containerfile_golangci_lint too
 
 .PHONY: lint
 lint: $(GOLANGCI_LINT_CACHE_DIR) container_composer_golangci_built.info
+	go fmt ./...
```

Comment:
Might need a specific go version here, sometimes between major versions fmt works differently, and people might have different default go versions than the one used by `tools/prepare-source`

Reply:
Nice catch, thanks!